src/codegate/pipeline/pii/pii.py

```diff
@@ -172,13 +271,13 @@ async def process_chunk(
         current_pos = 0
         result = []
         while current_pos < len(content):
-            start_idx = content.find("<", current_pos)
+            start_idx = content.find("#", current_pos)
```

Comment:
*nit:* I would use the instance variables, I don't see much use for them otherwise.
```suggestion
            start_idx = content.find(self.marker_start, current_pos)
```